tests/fixtures/json_validation/swagger.yaml

```diff
@@ -67,3 +67,16 @@ paths:
           description: User object
           schema:
             $ref: '#/definitions/User'
+
+  /nullable_default:
```

Comment:
Is this endpoint actually tested? The added test doesn't seem to call it?

Reply:
It's the loading of the spec, since the validation of the spec itself failed.